packages/ibm-products/src/components/SidePanel/_side-panel.scss

```diff
@@ -87,6 +86,9 @@ $action-set-block-class: #{$pkg-prefix}--action-set;
         border-right: 1px solid $decorative-01;
       }
     }
+    .#{$block-class}__actions-container {
+      width: 100%;
+    }
```

Comment:
Before: the actions container is the "same size" as the side panel. 
- this worked when using absolute values like`rems` ("Make the side panel and actions container 20rems wide"), 
- but broke when using relative values like `%` ("Make the side panel 75% the width of the page, and the actions container 75% of the side panel"). This issue only happens where `size` is `2xl`.

After: the actions container is a static 100% the width of the side panel.